README.md

```diff
@@ -1 +1,156 @@
+			'</div>'
+		].join(''),
+		scope: {
+            handle: '@'
+        },
+        controller: function ($scope, $timeout) {
+            $timeout(function () {
+                $scope.handle = 'angularjs'
+            }, 5000);
+        },
+		restrict: 'E'
+	};
+}
+
+angular
+	.module('app')
+	.directive('twitterCard', TwitterCard);
+```
+
+This will set our Twitter handle to "angularjs" after 5 seconds.
+
+## controllerAs
+
+As you've learned already, the best practise is in fact to use the `controllerAs` syntax. But we're passing a function through as our controller, so how are we going to tell Angular that we want to use `controllerAs`? You guessed it - the `controllerAs` property.
+
+Much like our `ng-controller`, where we use `SomeController as some`, we use `controllerAs` and put it's value as `some` (or whatever you want to call it).
+
+```js
+function TwitterCard() {
+	return {
+		template: [
+			'<div class="twitter">',
+				'<a href="https://twitter.com/{{ twitter }}">Follow @{{ twitter }} on Twitter!</a>',
+				'<button ng-click="ctrl.changeHandle()">Change Handle</button>',
+			'</div>'
+		].join(''),
+		scope: {
+            handle: '@'
+        },
+        controller: function ($scope) {
+            // $scope.handle === 'billgates'
+
+            this.changeHandle = function () {
+                $scope.handle = 'angularjs';
+            };
+        },
+        controllerAs: 'ctrl',
+		restrict: 'E'
+	};
+}
+
+angular
+	.module('app')
+	.directive('twitterCard', TwitterCard);
+```
+
+As you can see here, we're setting our controller as `ctrl`. This means that anything we attach to `this` in the controller will be accessible via the `ctrl` object in our template - exactly like our controllers before!
+
+You might have noticed how `handle` is still on our `$scope` object - this is because it's passed through as a scope property. We'll cover how to get that into `this` soon.
+
+## Existing Controllers
+
+Instead of a function, we can pass through a string to the `controller` property as well - much like what we do with `ng-controller`. This means we can use an existing controller as our directive's controller. It also means that we can remove the `controllerAs` property if we're using a string, as we can do `'SomeController as some'`.
+
+```html
+<twitter-card handle="billgates"></twitter-card>
+```
+
+```js
+function TwitterController($scope) {
+	// $scope.handle === 'billgates';
+}
+
+angular
+	.module('app')
+	.controller('TwitterController', TwitterController);
+
+function TwitterCard() {
+	return {
+		template: [
+			'<div class="twitter">',
+				'<a href="https://twitter.com/{{ twitter }}">Follow @{{ twitter }} on Twitter!</a>',
+				'<button ng-click="ctrl.changeHandle()">Change Handle</button>',
+			'</div>'
+		].join(''),
+		scope: {
+            handle: '@'
+        },
+        controller: 'TwitterController as ctrl',
+		restrict: 'E'
+	};
+}
+
+angular
+	.module('app')
+	.directive('twitterCard', TwitterCard);
+```
```

Comment:
Add a conclusion here, one or two lines wrapping up what the big takeaways of the README are